app/main.py

```diff
@@ -1,6 +1,32 @@
+import os
+from dotenv import load_dotenv
+import requests
+
+
+load_dotenv()
+BASIC_URL = os.getenv("BASIC_URL")
+API_KEY = os.getenv("API_KEY")
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(f"{BASIC_URL}/current.json", {
+        "key": API_KEY,
+        "q": CITY
+    })
```

Comment:
What if `BASIC_URL='google.com'`?

Reply:
Then BASIC_URL='google.com' should be added to .env. I think I don't get the question. Please clarify. Thank you

Reply:
This is redundant variable. It must be constant. BASIC_URL = "http://api.weatherapi.com/v1"